modules/flowable-process-validation/src/main/java/org/flowable/validation/validator/impl/ExternalInvocationTaskValidator.java

```diff
@@ -32,7 +32,7 @@ protected void validateFieldDeclarationsForEmail(org.flowable.bpmn.model.Process
         boolean textOrHtmlDefined = false;
 
         for (FieldExtension fieldExtension : fieldExtensions) {
-            if ("to".equals(fieldExtension.getFieldName())) {
+            if ("to".equals(fieldExtension.getFieldName()) || "cc".equals(fieldExtension.getFieldName()) || "bcc".equals(fieldExtension.getFieldName())) {
```

Comment:
Maybe this is just my coding preferences but it seems that `fieldExtension.getFieldName()` is executed 7 times in this loop; wouldn't something like this be cleaner?
````
        for (FieldExtension fieldExtension : fieldExtensions) {
            switch (fieldExtension.getFieldName()) {
                case "to":
                case "cc":
                case "bcc": {
                    toDefined = true;
                    break;
                }
                case "html":
                case "htmlVar":
                case "text":
                case "textVar": {
                    textOrHtmlDefined = true;
                }
            }
        }
````

Reply:
@dbmalkovsky  I did above changes.